ouroboros-consensus/src/Ouroboros/Consensus/Storage/LedgerDB/InMemory.hs

```diff
@@ -746,7 +743,7 @@ ledgerDbMaxRollback :: (GetTip (l ValuesMK), GetTip (l EmptyMK)) => LedgerDB l -
 ledgerDbMaxRollback LedgerDB{..} =
   let
     old = fromIntegral . AS.length <$> ledgerDbCheckpoints
-    new = fromIntegral $ AS.length $ dbChangelogVolatileCheckpoints ledgerDbChangelog
+    new = fromIntegral $ AS.length $ changelogVolatileStates ledgerDbChangelog
```

Comment:
Discussed with Nick: revert to `dbChangelogVolatileCheckpoints` so that we don't have to reach for the internals.